apps/decoder_csv/lib/decoder/csv.ex

```diff
@@ -22,7 +23,7 @@ defmodule Decoder.Csv do
             headers: nil,
             skip_first_line: false
 
-  defimpl Decoder, for: __MODULE__ do
```

Comment:
this requirement was just deprecated and is now implied, correct? this is still happening under the hood?

Reply:
yes, when calling defimpl in a module the "for:" is not needed